pkg/sypgp/sypgp.go

```diff
@@ -486,8 +493,10 @@ func (keyring *Handle) GenKeyPair(keyServiceURI string, authToken string) (*open
 
 	fmt.Printf("Generating Entity and OpenPGP Key Pair... ")
 
-	entity, err := keyring.genKeyPair(name, comment, email, passphrase)
+	entity, err := keyring.genKeyPair(name, comment, email, passphrase, bitLen)
 	if err != nil {
+		// Print the missing newline if there’s an error
+		fmt.Printf("\n")
```

Comment:
the error is printed to the stdout?

Reply:
This only adds the newline after:

```go
fmt.Printf("Generating Entity and OpenPGP Key Pair... ") // line 487, or 494
```

In case the key gen fails, this just adds the newline so the return err is printed at the next line.

Reply:
Why not adding \n into the actual output then?

Reply:
Nicer output, the `openpgp.NewEntity` can take some time to generate the key, and it looks nicer with the `... done` imo.